test/test_view_ops.py

```diff
@@ -1493,6 +1493,12 @@ def test_view_all_dtypes_and_devices(self, device):
             x = torch.tensor([[1, 2], [3, 4], [5, 6]], dtype=dt, device=device)
             self.assertEqual(x.view(6).shape, [6])
 
+    @onlyCPU
+    def test_conj_neg_view_numpy_error(self, device):
+        self.assertRaisesRegex(RuntimeError, "has conjugate bit set", lambda: torch.tensor([1+2j]).conj().numpy())
+        self.assertRaisesRegex(RuntimeError, "has negative bit set", lambda: torch.tensor([1+2j]).conj().imag.numpy())
+        self.assertRaisesRegex(RuntimeError, "not supported for conjugate view tensors", lambda: torch.tensor([1+2j]).conj().view(torch.float64))
+        self.assertRaisesRegex(RuntimeError, "not supported for tensors with negative bit set", lambda: torch.tensor([1+2j]).conj().imag.view(torch.int32))
```

Comment:
To avoid the lines are too long problem, you can use these as context manager:
```python
with self.assertRaisesRegex(RuntimeError, "not supported for tensors with negative bit set"):
    torch.tensor([1 + 2j]).conj().imag.view(torch.int32)
```